resources/zib/terminology/ValueSet-GCS_EyesCodelijst-2.16.840.1.113883.2.4.3.11.60.40.2.12.8.1--20200901000000.xml

```diff
@@ -1,12 +1,21 @@
 <ValueSet xmlns="http://hl7.org/fhir">
    <id value="2.16.840.1.113883.2.4.3.11.60.40.2.12.8.1--20200901000000"/>
    <meta>
+      <source value="http://decor.nictiz.nl/fhir/4.0/zib2020bbr-"/>
       <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
       <!--<profile value="http://hl7.org/fhir/4.0/StructureDefinition/ValueSet"/>-->
+      <tag>
+         <system value="http://hl7.org/fhir/FHIR-version"/>
+         <code value="4.0.1"/>
+      </tag>
    </meta>
+   <text>
+      <status value="empty"/>
+      <div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div>
+   </text>
    <extension url="http://hl7.org/fhir/StructureDefinition/resource-effectivePeriod">
       <valuePeriod>
-         <start value="2020-09-01T00:00:00+00:00"/>
+         <start value="2020-09-01T00:00:00Z"/>
```

Comment:
should we include this in the ticket/release notes as well? Not so important, so we can skip I guess.

Reply:
As a general comment perhaps?